packages/plugin-react/src/index.js

```diff
@@ -1,48 +1,33 @@
 module.exports = class BugsnagReactPlugin {
-  constructor (React = window.React) {
-    if (!React) throw new Error('cannot find React')
-    this.React = React
+  constructor (...args) {
     this.name = 'react'
+    this.lazy = args.length === 0 && !window.React
```

Comment:
Is there a use case where I'd want to use deferred loading even if `window.React` is defined?

Reply:
I supposed you could specifically pass `undefined` if you really wanted to do that

Reply:
I don't think passing `undefined` would achieve what you want there, because in that case `args.length = 1` so `this.lazy` would be `false`.

But I'm not sure, hopefully this a road we don't need to go down. If `window.React` is defined but that is not the reference to React that is desired, that implies the existence of _another_ version of React on the page.

If we want to support defering in that case, then I think we would need to special case the value of `args[0]` to allow something like `{ lazy: true }` or `'LAZY'`. To disambiguate.